package.json

```diff
@@ -1,17 +1,21 @@
 {
-  "name": "n8n",
+  "name": "n8n-monorepo",
   "version": "0.200.0",
   "private": true,
   "homepage": "https://n8n.io",
+  "engines": {
+    "node": ">=16.17",
```

Comment:
> We suggest using [Node.js](https://nodejs.org/en/) version 16 for development purposes.

If 16.17 is the minimum required, then `CONTRIBUTING.md` should specify the minor.